R/RaMP.R

```diff
@@ -0,0 +1,185 @@
+#' @importClassesFrom DBI DBIConnection
+#'
+#' @importFrom methods setClassUnion
+#' 
+#' @noRd
+setClassUnion("DBIConnectionOrNULL", c("DBIConnection", "NULL"))
+
+setClass(
```

Comment:
Note: here I'm storing directly the database connection within the object. Alternatively it would also be possible to store just the `DBDriver`, database name and connection information (user, host etc). The `.dbcon` function below could then connect to the database upon request.